.gitignore

```diff
@@ -0,0 +1,7 @@
+venv/
+
+pycahe/
+
+.idea/
+
+*.sqlite3
```

Comment:
There is no need to add an empty line between each ignore rule in the `.gitignore`, add all rules without empty lines.